### PR TITLE
New configuration example for the K8s Prometheus Operator

### PR DESCRIPTION
**Pull request by genegr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ We will address these as soon as we can, but there are no specific SLAs.
...
 
-The stateless design of the exporter allows for easy configuration management as well as scalability for a whole fleet of Pure Storage arrays. Each time the OpenMetrics client scrapes metrics for a specific system, it should provide the hostname via GET parameter and the API token as Authorization token to this exporter.
+The stateless design of the exporter allows for easy configuration management as well as scalability for a whole fleet of Pure Storage arrays. The design follows almost completely the [multi-target-exporter](https://prometheus.io/docs/guides/multi-target-exporter/) pattern described in the Prometheus documentation, so the tool can be used to scrape multiple FlashArrays from a single instance or just act the front-end for a single FlashArray. Each time the OpenMetrics client scrapes metrics for a specific system, it should provide the target array name or IP address via GET parameter to this exporter.
```
Comment (chrroberts-pure) [inline README.md:16]: We should state that the authorization tokens can still be passed via Bearer Authorization token in the GET request.

**Review by chrroberts-pure — DISMISSED** [review]
LGTM, besides one small nit on wording change in the README.

File: README.md (hunk 1 above)
Comment (sdodsley) [reply README.md]: I edited this so remove the last sentence and promoted the Authentication section to a full header.
This should make the whole thing read cleaner

**Comment by chrroberts-pure** [comment]
approved.

**Review by sdodsley — APPROVED** [review]

**Pull request merged**